lib/util.js

```diff
@@ -282,27 +282,30 @@ const util = {
 
       // Mapping for copying Brave's Android resource into chromium folder.
       const copyAndroidResourceMapping = {
-        [androidIconSource]: androidIconDest,
-        [androidIconSource]: androidIconDestAdditional,
```

Comment:
that was my bad as we use map there and the value was replaced by a key, so basically it started to copy the latest only. Debug btw gets icons from a first location and release from a second, so debug was broken after that. I redid it to use an array of destination paths